### PR TITLE
21.04.21 in die Sammel-PR

### PR DESCRIPTION
**Pull request by ZeroPointMax**
Die Pfeile sind hier mal repariert und neuer Stoff kommt auch hinzu

### CODE REVIEW

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWS-KUSCHE/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ABWL-LEDER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ASM-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/DBS-DORENDORF/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/PRO-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWE-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/EMB-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/OOP-CASSELT/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- SWS-KUSCHE/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

File: SWS-KUSCHE/README.md [hunk 1]
```diff
@@ -579,3 +597,215 @@ zu 2.: Nutzung von Zertifikaten: PKI / Public Key Infrastructure
...
+
+# Sichere Programmierung
+
+## Angriffe
+
+- Datendiebstahl z.B. zum Verkauf der Daten
+- Datenzerstörung, -verschlüsselung, -veränderung z.B. zur Erpressung
+- Denial Of Service, also Ausfall
+- Resourcendiebstahl (Rechenzeit, Internet, Speicherplatz)
+- Motivation: "Ich bin so cool", wirtschaftliche Gründe (Spionage, Wettbewerbsvorteil, Rufschädigung,...); persönliche Gründe; politische, geheimdienstliche Absichten; Geld
```
Comment (Steve2955) [inline SWS-KUSCHE/README.md:790]:
```suggestion
- Motivation: *"Ich bin so cool"*, wirtschaftliche Gründe (Spionage, Wettbewerbsvorteil, Rufschädigung,...); persönliche Gründe; politische, geheimdienstliche Absichten; Geld
```

**Review by Steve2955 — APPROVED** [review]
Danke für die Mitschriften 👍

File: SWS-KUSCHE/README.md [hunk 2]
```diff
@@ -579,3 +597,215 @@ zu 2.: Nutzung von Zertifikaten: PKI / Public Key Infrastructure
...
+- arbeitet transparent und "on the fly"
+- sobald entsperrt, sind alle Daten lesbar
+- Zweck: z.B. Diebstahlschutz, Schutz vor fremden Betriebssystemen
+- schützt nicht vor Fremdzugriff in einer angemeldeten Sitzung, Malware, Fileshares
+
+![Disk Encryption](assets/disk_encryption.png)<!--width=400px-->
+
+## Unterschied zu File Encryption
+
+- File E. verschlüsselt nur den Inhalt, nicht die Metadaten
```
[the remark this answers is not in the document]
Comment (ZeroPointMax) [reply SWS-KUSCHE/README.md:652]: ups :D

File: SWS-KUSCHE/README.md (hunk 1 above)
Comment (ZeroPointMax) [reply SWS-KUSCHE/README.md:790]: 😂

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWS-KUSCHE/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ABWL-LEDER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ASM-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/DBS-DORENDORF/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/PRO-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWE-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/EMB-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/OOP-CASSELT/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- SWS-KUSCHE/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

**Pull request merged**